src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -37,13 +36,13 @@ object RuleUtils {
   /**
    * Get active indexes for the given logical plan by matching signatures.
```

Comment:
could you please update the description

Reply:
also the pre-requisite of ACTIVE indexes

Reply:
Thanks! I updated the comment. BTW seems active indexes are not pre-requisite for this function as we check the status with  `index => index.created` (`def created`: status == "ACTIVE")